docs/janitor-archive.md

```diff
@@ -0,0 +1,27 @@
+## Janitor-Archive
+
+- Default port: `9914/TCP`
+- [Dockerfile](https://github.com/jelmer/janitor/blob/main/Dockerfile_archive): `podman pull ghcr.io/jelmer/janitor/archive` ([Container Registry](https://github.com/jelmer/janitor/pkgs/container/janitor%2Farchive))
+- [Source](https://github.com/jelmer/janitor/blob/main/py/janitor/debian/archive.py)
+
+<!-- `python3 -m janitor.debian.archive` -->
+
+```console
+$ janitor-archive --help
```

Comment:
Please don't include help output here verbatim - this will just get out of date, and users can just invoke the command manually.

Reply:
I was under the impression that it was mostly standard to have the help output when looking at docs for a project.

I'm happy to drop, just didn't think the output would change a huge amount?

Reply:
Dropped

Reply:
The problem with this is that it adds to maintenance overhead - same goes for other things like docker compose config. Having outdated docs is worse than no docs. If I rename a parameter I don't want to have to update it in many different places. 

I'd be more amenable to this if this was auto-generated or automatically updated somehow.